build/Build.cs

```diff
@@ -18,7 +18,6 @@
 using static Nuke.Common.IO.PathConstruction;
 using static Nuke.Common.Tools.DotNet.DotNetTasks;
 
-[CheckBuildProjectConfigurations]
```

Comment:
What does removing this do?

Reply:
It's now obsolete in the new nuke version

```
/dev/NukeBuildComponents/build/Build.cs(21,2): warning CS0618: 'CheckBuildProjectConfigurationsAttribute' is 
obsolete: 'This attribute solely checks whether solutions have active build configurations for the build project, 
which leads to an error because the build project cannot be compiled again while executing. The sanity check is 
now integrated in the SolutionAttribute' [/dev/NukeBuildComponents/build/_build.csproj]
```